zebra/zebra_nb_config.c

```diff
@@ -2697,6 +2697,29 @@ int lib_interface_zebra_ipv6_router_advertisements_default_lifetime_destroy(
 	return NB_OK;
 }
 
+/*
+ * XPath: /frr-interface:lib/interface/frr-zebra:zebra/ipv6-router-advertisements/fast-retransmit
+ */
+int lib_interface_zebra_ipv6_router_advertisements_fast_retransmit_modify(
+	struct nb_cb_modify_args *args)
+{
+	struct interface *ifp;
+	struct zebra_if *zif;
+	bool fast_retransmit;
+
+	if (args->event != NB_EV_APPLY)
+		return NB_OK;
+
+	ifp = nb_running_get_entry(args->dnode, NULL, true);
+	zif = ifp->info;
+
+	fast_retransmit = yang_dnode_get_bool(args->dnode, NULL);
+
+	zif->rtadv.UseFastRexmit = fast_retransmit;
```

Comment:
In an earlier commit the bool return was not trusted and a `!!` was applied. Do we trust the lyd_value or not? :) libyang claims it will be 0 or 1.

Reply:
Well, I believe it 😄
But if we want an additional check, it's better to place it into `yang_dnode_get_bool` directly.

Reply:
Actually, according to C99:
```
When any scalar value is converted to _Bool, the result is 0 if the value compares equal to 0; otherwise, the result is 1.
```
And `bool` is just a macro that expands to `_Bool`. So we're safe here without any additional checks.

Reply:
I had thought I'd read that before as well.. In any case this means there are a lot of uses of unnecessary `!!`s in this PR...

Reply:
Right, we can clean this up separately.